src/parser/XMLParser.java

```diff
@@ -266,7 +270,7 @@ private static List<Edge> setEdges(Element el, List<Clock> clocks, List<BoolVar>
             }
 
             if (chan == null) {
-                throw new IllegalStateException("Requires a chan");
+                throw new IllegalStateException(edge + "is missing a channel");
```

Comment:
Will there be a space between `edge` and `"is missing a channel"` when this exception is thrown?